vaadin-side-nav-flow-parent/vaadin-side-nav-flow-integration-tests/src/test/java/com/vaadin/flow/component/sidenav/tests/SideNavIT.java

```diff
@@ -115,6 +115,24 @@ public void collapseSideNav_expandedStateSynchronized() {
         assertExpandedStateOnServer("print-side-nav-expanded-state", "false");
     }
 
+    @Test
+    public void addMatchingPathAliasToNonNavigableParent_itemActive() {
```

Comment:
Looks like these integration tests could be replaced with unit tests simply verifying the correctness of the `pathAliases` property when using the path alias API.

Reply:
Removed the integration tests.